app/services/Math.js

```diff
@@ -0,0 +1,15 @@
+const preciseOperation = f => (a, b, decimalDigits = 6) =>
```

Comment:
How did you decide that 6 decimals is precise? BitCoin e.g. has 8. THis might be appropriate to calculate prices for the scaled orders, but not in general IMO

Reply:
@sschiessl-bcp it is not precise in the meaning we used for. It is func to round numbers only.

For e.g. 
1.234567 -> 1.23456
1234567890.1234567890 -> 1234567890.123456